### DISCUSSION TITLE
"error":{"code":-32000,"message":"missing trie node  on archive mainnet node

### DISCUSSION THREAD

**Discussion by danlix2000** [ethereum-optimism/developers#471] [Application Development] [answered]
I used the latest pebble db snapshots to sync this node (mainnet-2024-05-09.tar.zst)

getting this issue.

> {"jsonrpc":"2.0","id":1,"error":{"code":-32000,"message":"missing trie node 70e121cfd31b92f8e8a8f3be71f6e1866ab8faa33ad45c29affd349f3b02674e (path ) state 0x70e121cfd31b92f8e8a8f3be71f6e1866ab8faa33ad45c29affd349f3b02674e is not available, not found"}}

op-geth service file

`ExecStart=/home/ubuntu/op-geth/build/bin/geth --datadir=/data/datadir --op-network=op-mainnet --http --http.port=8545 --http.addr=0.0.0.0 --authrpc.addr=localhost --authrpc.jwtsecret=./jwt.txt --verbosity=3 --rollup.sequencerhttp=https://mainnet-sequencer.optimism.io/  --syncmode=full --maxpeers=100 --rpc.batch-request-limit=50000 --rpc.batch-response-max-size=2500000000  --rollup.historicalrpc=http://1x.1x.1x.x:8546/ --gcmode archive --db.engine=pebble  --state.scheme=hash  --http.corsdomain=* --http.vhosts=* --nat=extip:x.x.x.x  --cache=13107 --rpc.gascap=250000000 --http.api=eth,web3,net,debug,txpool --syncmode=full`

this is my call.

```
curl --location 'http://xxxxxxx' \
--header 'Content-Type: application/json' \
--data '{"jsonrpc":"2.0","id":1,"method":"eth_call","params":[{"data":"0x23d841220000000000000000000000001f9090aae28b8a3dceadf281b0f12828e676c326000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000012000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000160000000000000000000000000000000000000000000000000000000000000000300000000000000000000000000000000000000000000000000000000000001c00000000000000000000000000000000000000000000000000000000000000240000000000000000000000000000000000000000000000000000000000000028000000000000000000000000000000000000000000000000000000000000002c00000000000000000000000000000000000000000000000000000000000000320000000000000000000000000000000000000000000000000000000000000001262616c616e63654f6628616464726573732900000000000000000000000000000000000000000000000000000000000000000000000000000000000000000025746f6b656e4f664f776e65724279496e64657828616464726573732c2075696e7432353629000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000012636c61696d61626c652875696e74323536290000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000020000000000000000000000009c7305eb78a432ced5c4d14cac27e8ed569a2e260000000000000000000000009c7305eb78a432ced5c4d14cac27e8ed569a2e2600000000000000000000000000000000000000000000000000000000000000020000000000000000000000005d5bea9f0fc13d967511668a60a3369fd53f784f0000000000000000000000009c7305eb78a432ced5c4d14cac27e8ed569a2e26","to":"0xbaa78247EB8D45a3BAb575C17024e445bEf1D966"},"0x693845a"]}'
```

**Comment by opfocus** [comment]
CC @Chomtana

**Reply by danlix2000** [reply]
I downloaded old leveldb (4 TB) archive snapshots and  started to sync a node.
executed above call. and it is working. Seems your new pebbledb snapshots have issues. 

> curl --location 'http://x.x.x.x' \
> --header 'Content-Type: application/json' \
> --data '{"jsonrpc":"2.0","id":1,"method":"eth_call","params":[{"data":"0x23d841220000000000000000000000001f9090aae28b8a3dceadf281b0f12828e676c326000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000012000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000160000000000000000000000000000000000000000000000000000000000000000300000000000000000000000000000000000000000000000000000000000001c00000000000000000000000000000000000000000000000000000000000000240000000000000000000000000000000000000000000000000000000000000028000000000000000000000000000000000000000000000000000000000000002c00000000000000000000000000000000000000000000000000000000000000320000000000000000000000000000000000000000000000000000000000000001262616c616e63654f6628616464726573732900000000000000000000000000000000000000000000000000000000000000000000000000000000000000000025746f6b656e4f664f776e65724279496e64657828616464726573732c2075696e7432353629000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000012636c61696d61626c652875696e74323536290000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000020000000000000000000000009c7305eb78a432ced5c4d14cac27e8ed569a2e260000000000000000000000009c7305eb78a432ced5c4d14cac27e8ed569a2e2600000000000000000000000000000000000000000000000000000000000000020000000000000000000000005d5bea9f0fc13d967511668a60a3369fd53f784f0000000000000000000000009c7305eb78a432ced5c4d14cac27e8ed569a2e26","to":"0xbaa78247EB8D45a3BAb575C17024e445bEf1D966"},"0x693845a"]}'
> {"jsonrpc":"2.0","id":1,"result":"0x00000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000001200000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000000a000000000000000000000000000000000000000000000000000000000000000c00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000000a000000000000000000000000000000000000000000000000000000000000000c0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"}

**Reply by opfocus** [reply]
The op-team has been informed of this issue and will give you feedback when the results are available.

**Reply by payne0420** [reply]
Can confirm, the snapshot provided is a fullnode not archive.

**Reply by opfocus** [reply]
> Can confirm, the snapshot provided is a fullnode not archive.
Your suspicions seem to be right, there is some discussion on discord
https://discord.com/channels/667044843901681675/1080862392281481246/1246079005824778301
https://discord.com/channels/667044843901681675/1080862392281481246/1246119003248656516
https://discord.com/channels/667044843901681675/1080862392281481246/1246123841202290709

**Comment by sbvegan** [comment, marked as answer]
Thank you for the flag. Our developer infrastructure team is reconfiguring the archive node to create a new archive snapshot. cc @felipe-op to close when the snapshot is uploaded
